src/stdune/console.ml

```diff
@@ -1,78 +1,130 @@
-module Display = struct
-  type t =
-    | Progress
-    | Short
-    | Verbose
-    | Quiet
-
-  let all =
-    [ ("progress", Progress)
-    ; ("verbose", Verbose)
-    ; ("short", Short)
-    ; ("quiet", Quiet)
-    ]
-end
+module Backend = struct
+  module type S = sig
+    val print : string -> unit
```

Comment:
Not related to this PR, but why do we need this function? It seems like `Pp` should be encouraged here instead.

Reply:
Yh, that's true actually. Let me have a look at this.

Reply:
Using `Pp.t` here is indeed better and allows to improve the code and formatting in other places, such as in `Report_error`, but it's a bigger change so I'll submit a separate PR.

Reply:
Done in #3242